content/sessions/keynote_open.md

```diff
@@ -6,6 +6,7 @@ tags:
   - keynote
 speakers: []
 draft: false
+videoId: Y4JmMXf55Ow
```

Comment:
From what I understand, we use the same `keynote_open` for both days, so we can't reference the second keynote video.
It's not a blocker for merging this :wink:

Reply:
Yep, I realized the same thing when doing the PR :grimacing: